docs/2.1.a-bug.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html translate="no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Flexbug 2.1.a (bug)</title>
+    <style>
```

Comment:
> One thing that might not be clear from my CodePen examples is that they all use normalize.css and autoprefixer.

I did not realize this. Thanks for bringing this to my attention.

> If you look at the source of one of them (in debug mode), you can see the final CSS used:
> `view-source:https://cdpn.io/philipwalton/debug/xbRpMe/PNkvYKJwNdXA`

This link does NOT work for me - it gives me the source to the page which states: **This debug view expired.** However, I was able to clone your pen on codepen, and I was able to view the source of my pen in debug mode. From the source code, I see that codepen:

- is including an [outdated](https://classic.yarnpkg.com/en/package/normalize.css#5-0-0-october-3-2016-) version of `normalize.css` from a Content Delivery Network (CDN) i.e. [cdnjs.com](https://cdnjs.com/libraries/normalize/).
- automatically provides `-webkit-` CSS prefixes. From the source, I could NOT deduce how this is done using `autoprefixer` - I presume that `autoprefixer` is executed during a Continuous Integration/Delivery (CI/CD) process.

> What do you think would be the best way to handle this? Do you know if you can run any build steps as part of the gh-pages deploy?

I am a novice on gh-pages myself - not aware of the CI/CD process yet. I am aware that it utilizes `jekyll` as a static site generator tool. I can learn more about the CI/CD process and find out how to implement this requirement. For now, I have manually included the following - copied from the codepen sources:

- `normalize.css` as a CDN link.
   - I used the minified version ([`normalize.min.css`](https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css)). Let me know if you think we should use the non-minified version ([`normalize.css`](https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css)). 
   - I used [`v5.0.0`](https://classic.yarnpkg.com/en/package/normalize.css#5-0-0-october-3-2016-). Let me know if you think we should use the latest i.e. [`v8.0.1`](https://classic.yarnpkg.com/en/package/normalize.css#8-0-1-november-4-2018-).
   - I did NOT use the Subresource Integrity (SRI). Let me know if you think we should use SRI.
- `-webkit-` prefixes.

> If not, it's probably fine to drop autoprefixer. I'm mean, it looks like the -ms- prefixes for IE 10 are no longer being added by CodePen's autoprefixer config anyway (though for historical reasons it'd be nice if they could be).

- I don't see `-ms-` prefixes (for IE) too.
- I don't see `-moz-` prefixes (for older versions of Mozilla Firefox) either.

Reply:
To get the IE prefixes, you can run autoprefixer and make sure to specify a [browserlist](https://github.com/browserslist/browserslist) config that includes IE10 and IE11. If you prefer a web interface, you can also do it here: https://autoprefixer.github.io/

If it's easier to just do it manually for now, that's fine with me. The browserlist [config](https://browserl.ist/?q=last+2+versions%2C+firefox+ESR%2C+ie+10) I'd recommend using is: `last 2 versions, firefox ESR, ie 10`

Reply:
Re: normalize.css, I think it makes sense to use the latest. And I'm not particularly concerned about integrity since this is just a demo and it's a CSS file.

Reply:
@philipwalton
1. The CSS has been run thru the `autoprefixer` web interface using the recommended browserlist config. Pasted the resulting CSS into the corresponding source-code files.
1. `normalize.css` has been updated to use `v8.0.1`.